src/components/rating/rating.tsx

```diff
@@ -227,6 +226,24 @@ export class Rating implements LabelableComponent, FormComponent, InteractiveCom
     this.calciteRatingChange.emit({ value });
   }
 
+  private onKeyboardPressed = (event: KeyboardEvent): void => {
+    if (!this.required && (event.key === "Enter" || event.key === " ")) {
+      //* prevent default event handling only for 'Enter' and 'Space' keys
+      event.preventDefault();
+      this.updateValue(0);
+    }
+  };
+
+  private onFocusChange = (index: number): void => {
```

Comment:
Can you rename `index` to something more meaningful in the context of this method?